src/environmentd/src/deployment/preflight.rs

```diff
@@ -100,7 +100,8 @@ pub async fn preflight_legacy(
                 .await?);
             }
             Err(e) => {
-                return Err(anyhow!(e).context("Catalog upgrade would have failed with this error"))
+                tracing::warn!(error = %e, "catalog upgrade would have failed");
```

Comment:
Should we upgrade this to an error log? I think this is always unexpected and requires manual intervention.

Reply:
I think it'd be noisy to send this to Sentry? Agreed that it's unexpected and requires manual intervention, but we have good alerting elsewhere for envds that are in crashloops, and sending this error to Sentry on every error seemed like it'd be pretty spammy.